docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -16,22 +16,19 @@ When an issue comes in,
 
 # Assigning reviewers for PRs
 
-* A PR should have,
-  * **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)
-  * **all reviewers should be selected as `assignees`**
-  
-  > Duplicating `reviewers` as `assignees` is needed because GitHub filter syntax cannot filter by reviewers yet.
+* A PR should have **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)<br>
```

Comment:
This line should be formatted similar to [line 36](https://github.com/oss-generic/process/pull/62/files#diff-8663d1e7fd254bc258c2c7c8b8d8f337R36).